src/pseudopeople/noise_functions.py

```diff
@@ -142,6 +142,23 @@ def apply_do_not_respond(
 #     return dataset_data
 
 
+def duplicate_with_guardian(
```

Comment:
The implementation of this function will be in a future PR but for now it does nothing but allows us to run tests.